mattermost-10.2.advisories.yaml

```diff
@@ -579,6 +579,11 @@ advisories:
             componentType: apk
             componentLocation: /.PKGINFO
             scanner: grype
+      - timestamp: 2024-12-09T14:41:29Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-version-not-used
+          note: 'This vulnerability relates to versions of mattermost prior to v7.4.0, which is several releases old. Several sources tie the credit of the individual contributing to the remediation, the custom mattermost CVE ID (MMSA-2022-00124) and the CVE ID found in the National Vulnerability Database which leads to the conclusion this is caused by an overly broad CPE configuration on the NVD record causing all versions of mattermost to become flagged. '
```

Comment:
I don't mean to be nit-picky, but I'm not sure what this sentence is saying exactly:

>Several sources tie the credit of the individual contributing to the remediation, the custom mattermost CVE ID (MMSA-2022-00124) and the CVE ID found in the National Vulnerability Database which leads to the conclusion this is caused by an overly broad CPE configuration on the NVD record causing all versions of mattermost to become flagged.

The part about broad CPE configuration is right, but I'm not sure how the phrases before that relate to that. I think we could just say that the CPE configuration has no version constraints, so it matches all version of mattermost, even though the vulnerability was fixed back in the 7.x versions.

Reply:
I think I made everything murky by trying to explain what I wrote in detail here. But it has no place in an advisory so I removed and simplified.